addons/website_slides/static/src/xml/website_slides_course_join.xml

```diff
@@ -0,0 +1,12 @@
+<templates>
+    <t t-name="slide.course.join">
```

Comment:
I think we can name this file channel_management.xml so that we can put other small templates related to channel if more of them come (like slide_management.xml) .